octokit-project/src/main/java/org/example/calculator/Calculator.java

```diff
@@ -0,0 +1,15 @@
+package org.example.calculator;
+
+public class Calculator {
+
+    public static void main(String[] args) {
```

Comment:
[Severity: Minor] The main method is a good starting point, but it would be better to separate the arithmetic operations into their own methods.
Consider creating methods like 'add(int a, int b)' to perform the addition. This will make your code more modular and reusable.
Rationale: Using methods for specific tasks improves code organization and allows for easier testing and maintenance.
Reference: https://www.oracle.com/java/technologies/javase/tutorial/java/javaOO/classes/methods.html